CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ description: |-
 
 # VKCS Provider's changelog
 
+#### v0.2.0 (unreleased)
+- Fix error of not finding an object for sharedfilesystem_sharenetwok datasource
```

Comment:
'fix error' - does it mean we could fix smth else?

Fix finding SharedNetwork object via datasource
Fix searching of shared network with datasource

does not this look better?

Reply:
I just sticked to the way we done before, but it is better, yup.